lib/statsig/application.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Statsig.Application do
+  use Application
+  require Logger
+
+  def start(_type, _args) do
+    Logger.error("Starting Statsig application")
+    Logger.error("All running applications: #{inspect(Application.started_applications())}")
+    Logger.error("Start type: #{inspect(_type)}")
+    Logger.error("Process dictionary: #{inspect(Process.get())}")
+    Logger.error("System flags: #{inspect(System.get_env())}")
+    Logger.error("Full stacktrace:", %{
+      stacktrace: try do
+        throw(:trace)
+      catch
+        :trace -> __STACKTRACE__
+      end
+    })
```

Comment:
likely for debugging, but remember to remove this. 
Also, stack traces might not show what you think they will, since erlang has tail call optimization.